fetchData.js

```diff
@@ -0,0 +1,71 @@
+class Digimon {
+    constructor(element, name, img, level) {
+        this.element = element;
+        this.name = name;
+        this.img = img;
+        this.level = level;
+        this.element.classList.add('digimon');
+        if (this.level == 'In Training' || this.level == 'Training'){
```

Comment:
This looks great. Some feedback to clean up your code:
Instead of having this large if statement to add the classname, how about adding a data attribute `data-level`? You can use setAttribute() to set `data-level` to this.level, and in your CSS can you use the CSS selector `[data-level="..."] { ... }` to set the styles of that specific level.